app/navigation/Navigation.scala

```diff
@@ -361,31 +356,28 @@ class Navigation @Inject()(dateTimeHelper: DateTimeHelper,
   }
 
   protected[navigation] def reverseRouteForAppealStartPage(userRequest: UserRequest[_]): Call = {
-    if (userRequest.answers.getAnswer[Boolean](SessionKeys.isObligationAppeal).isDefined) {
+    if (userRequest.answers.getAnswer[Boolean](SessionKeys.isFindOutHowToAppeal).isDefined) {
       routes.YouCanAppealPenaltyController.onPageLoad()
     } else {
       Call("GET", appConfig.penaltiesFrontendUrl)
     }
   }
 
   protected[navigation] def reverseRouteForHonestyDeclaration(userRequest: UserRequest[_]): Call = {
-    if (userRequest.answers.getAnswer[Boolean](SessionKeys.isObligationAppeal).isDefined) {
+    if (userRequest.answers.getAnswer[Boolean](SessionKeys.isFindOutHowToAppeal).isDefined) {
       routes.AppealStartController.onPageLoad()
     } else {
       routes.ReasonableExcuseController.onPageLoad()
     }
   }
 
   protected[navigation] def reverseRouteForUploadEvidenceQuestion(userRequest: UserRequest[_], mode: Mode): Call = {
-    if (userRequest.answers.getAnswer[Boolean](SessionKeys.isObligationAppeal).isDefined) {
-      routes.AppealAgainstObligationController.onPageLoad(mode)
-    } else {
       routes.OtherReasonController.onPageLoadForWhyReturnSubmittedLate(mode)
-    }
+
   }
 
   protected[navigation] def reverseRouteForMakingALateAppealPage(userRequest: UserRequest[_], mode: Mode, jsEnabled: Boolean): Call = {
-    if (userRequest.answers.getAnswer[Boolean](SessionKeys.isObligationAppeal).isDefined) {
+    if (userRequest.answers.getAnswer[Boolean](SessionKeys.isFindOutHowToAppeal).isDefined) {
       reverseRoutingForUpload(userRequest, mode, jsEnabled)
     } else {
```

Comment:
remove lines 300 to 302 and 392

Reply:
Essentially yes (the line numbers maybe off but the `if (isObligationAppeal/isFindOutHowToAppeal...` part is no longer required only users on an reasonable excuse journey can access the late appeal page so no need for the if statement